submariner/fine-grained-connectivity.md

```diff
@@ -0,0 +1,86 @@
+# Fine Grained Connectivity
+
+## Summary
+
+Submariner currently provisions full mesh topology by default between the participating clusters.
+For some use-cases, we need fine-grained connectivity options. This proposal covers only the topology
+aspect of connectivity.
+
+This proposal aims to address the below use-case
+
+Hub-Spoke Topology -  Here only the client can talk to the server but not to the other clients to
```

Comment:
Can you please clarify which is Client and which is Server component in the Hub-Spoke topology. Also, include notes if Hub can talk to Spoke or not to avoid ambiguation.

Reply:
The hub and spoke will be a special case of clustersets were only two clusters are present,  no specific server will be marked as a hub or a spoke with this proposal. So they can talk in both ways. A cluster which is a part of multiple clustersset will be a hub logically but it is not explicit. I can add the details based on how we conclude on other comments.

Reply:
Done.